app/assets/javascripts/geoblacklight/viewers/index_map.js

```diff
@@ -1,4 +1,7 @@
 //= require geoblacklight/viewers/map
+var colorDefault = "#1eb300",
```

Comment:
I'd have to look into it further, but the [GBL wiki references this file for configuring Leaflet customizations](https://github.com/geoblacklight/geoblacklight/wiki/Adding-Leaflet-control-customization-to-GeoBlacklight#configure-your-settings), so I think it's at least worth asking the core team for advice.

FWIW, I do like Keith's suggestion to allow for customization of the entire style object.

Reply:
I figured out how to call it from the settings file this morning so I just pushed a commit that does that. I was able to find where they did it for the current settings and just pull over that line of code to do it for the index map. Now it calls all styles from the settings file. The only thing is that you have to define each aspect for every style group (default, unavailable, selected) so you're repeating a few things like stroke weight and radius but I think that's a decent trade off for being able to set those styles in a single place where anyone can get in and edit even if they aren't super knowledgable with javascript.

Reply:
This is a big improvement. We can always consider using YAML's merge capability as is currently being done with the [`opacity_control` setting](https://github.com/geoblacklight/geoblacklight/blob/v1.9.1/lib/generators/geoblacklight/templates/settings.yml#L83), if we wanted to simplify some of the shared defaults across these three styles and think it will be a common use case to keep the weight and radius consistent across the three states.

Reply:
I think it would be worth doing because, as it was before, the radius and weight were being used across all states from a single place. I'll work on getting this into place.

Reply:
Just an update on this, it is now using the YAML merge like the opacity control. I have all of the styles going into Default (like it was when it was all just in the javascript) and then default is added to the other styles, with the color being overwritten only since it is specifically defined in each of those.